spec/controllers/api/sequences/sequences_create_spec.rb

```diff
@@ -10,11 +10,16 @@
 
     it 'creates a new sequences for a user' do
       sign_in user
-      inputs = {name: "Scare Birds",
+      input = {name: "Scare Birds",
                 steps: [{
-                    message_type: 'move_rel'
-                  }]}
-      post :create, inputs
+                    message_type: 'move_rel',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.